src/commands/image/imageSource/buildImageInAzure/ImageNameStep.ts

```diff
@@ -35,19 +41,35 @@ export class ImageNameStep extends AzureWizardPromptStep<BuildImageInAzureImageS
 
         return undefined;
     }
+
+    static generateSuggestedImageName(repositoryName: string = 'hello-world'): string {
+        const tag: string = getTimestampTag();
+        return repositoryName.slice(0, maxImageNameLength - (tag.length + 1)) + ':' + tag;
```

Comment:
Why are you referring to the names as a repositoryName? Isn't it an image name?

Reply:
What's passed is a repository name, what comes out is an image name

Reply:
Okay yeah I'm pretty sure that the part coming in is the repository name:

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/f21f90c7-8314-403b-9921-3968bcaf841e)

and what comes out is the actual image name

Reply:
Ah yeah. Okay, fair enough.